table/tables/tables.go

```diff
@@ -104,7 +103,6 @@ func NewTable(tableID int64, tableName string, cols []*column.Col, alloc autoid.
 		indexPrefix:  string(genTableIndexPrefix(tableID)),
 		alloc:        alloc,
 		Columns:      cols,
-		state:        model.StatePublic,
```

Comment:
I find t.state is set here: https://github.com/pingcap/tidb/blob/master/table/tables/tables.go#L93

But it is not related with meta. So what is meta mean?